### PR TITLE
wget : add some options

### PR DESCRIPTION
**Pull request by smzm**
Options  __-P__ , __-i__ , __-b__ , __--user --pasword__ , __-tries__ , __--limit-rate__ added.

- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/wget.md [hunk 1]
```diff
@@ -11,26 +11,26 @@
...
 
-- Download a single web page and all its resources (scripts, stylesheets, images, etc.):
+- Download a single web page and all its resources with 3-second intervals between requests (scripts, stylesheets, images, etc.):
 
-`wget --page-requisites --convert-links {{https://example.com/somepage.html}}`
-
-- Download a full website, with 3-second intervals between requests:
-
-`wget --mirror --page-requisites --convert-links --wait=3 {{https://example.com}}`
+`wget --page-requisites --convert-links --wait=3 {{https://example.com/somepage.html}}`
```
Comment (mebeim) [inline pages/common/wget.md]: I don't know how much sense this example makes. Who would want to wait 3 seconds between each request when dumping a web page and its resources?

File: pages/common/wget.md [hunk 2]
```diff
@@ -11,26 +11,26 @@
...
 
 - Download all listed files within a directory and its sub-directories (does not download embedded page elements):
 
 `wget --mirror --no-parent {{https://example.com/somepath/}}`
 
-- Download the contents of an URL via authenticated FTP:
+- Limit the download speed with specific number of retry:
```
Comment (mebeim) [inline pages/common/wget.md]:
```suggestion
- Limit the download speed and the number of retries:
```

File: pages/common/wget.md [hunk 3]
```diff
@@ -11,26 +11,26 @@
...
+
+`wget --limit-rate={{300k}} -tries={{100}} {{https://example.com/somepath/}}`
```
Comment (mebeim) [inline pages/common/wget.md]:
```suggestion
`wget --limit-rate={{300k}} --tries={{100}} {{https://example.com/somepath/}}`
```
Typo.

File: pages/common/wget.md [hunk 4]
```diff
@@ -11,26 +11,26 @@
...
+
+- Download a file from a password protected site and FTP:
```
Comment (mebeim) [inline pages/common/wget.md]: This does not make much sense. FTP and HTTP Basic Auth are two different things and have different command-line options for user and password. You should pick one between the two (I'd suggest keeping FTP as it is more common).

File: pages/common/wget.md [hunk 5]
```diff
@@ -11,26 +11,26 @@
...
 
-`wget --ftp-user={{username}} --ftp-password={{password}} {{ftp://example.com}}`
+`wget --user={{username}} --password={{password}} {{https://example.com}}`
 
 - Continue an incomplete download:
 
 `wget -c {{https://example.com}}`
 
-- Enable quiet mode to suppress output:
+- Download all URLs stored in a text file to specific directory:
```
Comment (mebeim) [inline pages/common/wget.md]:
```suggestion
- Download all URLs stored in a text file to a specific directory:
```
Typo.

File: pages/common/wget.md (hunk 4 above)
Comment (smzm) [reply pages/common/wget.md]: Specify the user and assword work for both FTP and HTTP file. These parameters can be overridden using the --ftp-user and --ftp-password options for FTP connections and the --http-user and --http-password options for HTTP connections.

Comment (mebeim) [inline pages/common/wget.md]:
```suggestion
- Download a file from an HTTP server using Basic Auth (also works for FTP):
```
Sounds better, what do you think?

Comment (mebeim) [reply pages/common/wget.md]: Oh, if those parameters are the same then it's cool. Should probably reword the example description though. Check my new comment below.

Comment (smzm) [reply pages/common/wget.md]: Yes :+1: ,much better.
thanks

**Pull request merged**